app/controllers/api/accounts_controller.rb

```diff
@@ -6,12 +6,16 @@ class Api::AccountsController < Api::BaseController
 
   def resources
     @accounts ||= if params[:user_id]
-      User.find(params[:user_id]).accounts.order(created_at: :desc).page(params[:page])
+      User.find(params[:user_id]).accounts
     else
-      Account.active.member.order(created_at: :desc).page(params[:page])
+      Account.active.member
```

Comment:
the new `decorate_query` method calls `with_paging` by default, so the resource method doesn't have to.